### PR TITLE
Update API docs.

### PR DESCRIPTION
**Pull request by agne6ka**
Ticket: https://app.asana.com/0/1201235440001052/1201264543489281

### CODE REVIEW

File: docs/API-reference.md [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+}
+```
+
+</details>
+
+## Instance basic operations
+
+<details open>
+<summary>
+    <strong>[ GET ]</strong> <code>/api/v1/instances/</code> <small>- list all instances</small>
```
Comment (patuwwy) [inline docs/API-reference.md:142]: please remove trailing slash

File: docs/API-reference.md [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+}
+```
+
+</details>
+
+## Host operations
+
+<details>
+<summary>
+    <strong>[ GET ]</strong>  <code>/api/v1/stream/log</code> <small>- stream all host logs</small>
```
Comment (patuwwy) [inline docs/API-reference.md:361]: `/api/v1/stream/log`

File: docs/API-reference.md [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+        "pressure": 30
+    }]
+}
+```
+
+</details>
+
+- POST /api/v1/stream/stdin​
+- GET /api/v1/stream/stderr​
+- GET /api/v1/stream/stdout​
```
Comment (patuwwy) [inline docs/API-reference.md:390]: - POST /api/v1/instance/:xx/stdin​
- GET /api/v1/instance/:xx/stderr
- GET /api/v1/instance/:xx/stdout

**Review by patuwwy — CHANGES_REQUESTED** [review]
see comments

File: docs/API-reference.md [hunk 4]
```diff
@@ -0,0 +1,390 @@
...
+```
+
+</details>
+
+<details open>
+<summary>
+    <strong>[ GET ]</strong>  <code>/api/v1/instance/:id/health</code> <small>- check status about instance health</small>
+</summary>
+
+<br> <strong>**Parameters**</strong>
```
Comment (a-tylenda) [inline docs/API-reference.md:252]: endpoint /health has no parameters

Comment (alicja-gruzdz) [reply docs/API-reference.md:252]: thanks Aga, I will correct it.

**Comment by MichalCz** [comment]
Aga, let's add the "official quick install guide" at top of readme, before the install from repo one-liner.

Something like this:

* Install node.js v16.x [for instance from nodesource](https://github.com/nodesource/distributions/blob/master/README.md)
* Install docker [for instance via get-docker](https://github.com/docker/docker-install/blob/master/README.md) - ideally we'll add `(optionally)` here when @pietrzakacper is done.
* Install STH from repo:

```sh
npm i -g @scramjet/sth
```

* Start STH:

```sh
transform-hub
```

**Comment by alicja-gruzdz** [comment]
> Aga, let's add the "official quick install guide" at top of readme, before the install from repo one-liner.
> 
> Something like this:
> 
> * Install node.js v16.x [for instance from nodesource](https://github.com/nodesource/distributions/blob/master/README.md)
> * Install docker [for instance via get-docker](https://github.com/docker/docker-install/blob/master/README.md) - ideally we'll add `(optionally)` here when @pietrzakacper is done.
> * Install STH from repo:
> 
> ```shell
> npm i -g @scramjet/sth
> ```
> 
> * Start STH:
> 
> ```shell
> scramjet-transform-hub
> ```

thanks :) @a-tylenda is adding it in https://github.com/scramjetorg/transform-hub/pull/137

File: packages/cli/src/lib/commands/host.ts [hunk 5]
```diff
@@ -7,16 +7,33 @@ export const host: CommandDefinition = (program) => {
...
+    * Log: version number
+    */
     hostCmd
         .command("version")
         .description("get version")
         .action(async () => displayEntity(program, getHostClient(program).getVersion()));
 
+    /**
+    * Command `si host logs`
+    * Show whole logs form host
```
Comment (a-tylenda) [inline packages/cli/src/lib/commands/host.ts:22]:
```suggestion
    * Show whole logs from host
```

File: packages/cli/src/bin/index.ts [hunk 6]
```diff
@@ -31,12 +31,33 @@ const errorHandler = (err: ClientError) => {
...
+ * Start commander using defined config {@link Apple.seeds}
+ */
 (async () => {
     const conf = getConfig();
 
+    /**
+     * Commands
+     * ```
+     * pack [options] [<directory>]
+     * host [command]                something
```
Comment (a-tylenda) [inline packages/cli/src/bin/index.ts:44]:
```suggestion
     * host [command]                operations on host
```

File: packages/cli/src/lib/commands/host.ts [hunk 7]
```diff
@@ -7,16 +7,33 @@ export const host: CommandDefinition = (program) => {
         .command("host [command]")
         .description("something");
 
+    /**
+    * Command `si host version`
+    * Get STH version number form package.json file
```
Comment (a-tylenda) [inline packages/cli/src/lib/commands/host.ts:12]:
```suggestion
    * Get STH version number from package.json file
```

File: packages/cli/src/lib/output.ts [hunk 8]
```diff
@@ -2,6 +2,17 @@
 import { Response, ResponseStream } from "@scramjet/api-client";
 import { Command } from "commander";
 
+/**
+ * Command form commander contains obj with whole params, that user provide to the console.
```
Comment (a-tylenda) [inline packages/cli/src/lib/output.ts:6]:
```suggestion
 * Command from commander contains obj with whole params, that user provides to the console.
```

**Review by a-tylenda — APPROVED** [review]

File: docs/API-reference.md (hunk 3 above)
Comment (agne6ka) [reply docs/API-reference.md]: Done

**Review by patuwwy — APPROVED** [review]

**Pull request merged**